app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    person_list = [Person(person["name"], person["age"]) for person in people]
+
+    for person in person_list:
+
+        for human in people:
+
+            if person.name == human["name"]:
+                wife = human.get("wife")
+                husband = human.get("husband")
+
+                if wife:
+                    person.wife = Person.people[wife]
+                elif husband:
+                    person.husband = Person.people[husband]
```

Comment:
`human` and `person` sound like the same so use more understandable variables like `person_data` and `person_instance`